dashboard/test/ui/features/star_labs/musiclab/musiclab_drag_block.feature

```diff
@@ -24,3 +24,8 @@ Scenario: Dragging play sound block
 
   # There should now be a music timeline entry.
   And element ".timeline-element" is visible
+
+Examples:
+  | url                                                       | test_name               |
+  | http://studio.code.org/s/allthethings/lessons/46/levels/2 | music lab script level  |
+  | http://studio.code.org/musiclab                           | music lab incubator     |
```

Comment:
should this be `/projectbeats`? Or are they equivalent?

Reply:
`/musiclab` redirects to `/projectbeats`.  I thought it better to use this URL, since the controller is still called `MusiclabController`, but I think this will actually not work since we also verify the URL we have landed on, [here](https://github.com/code-dot-org/code-dot-org/pull/52820/files#diff-ee17fbdda49240c2400fd785d8aef184bea12e30e20ec1cfe3e98bb191877576R7).  Will change.